### PR TITLE
Add split frontend and backend containers

### PR DESCRIPTION
**Pull request by KaemonIsland**
This PR updates the local app config, along with applies the [separate frontend](https://backstage.io/docs/deployment/docker#separate-frontend) changes from backstage.

The build process is the same, although a bit longer, as before.

To build local docker containers just follow the [running-locally](https://department-of-veterans-affairs.github.io/lighthouse-backstage/running-locally/) document or run:

```bash
docker compose -f .localdevcontainer/local-docker-compose.yml up --build
```

This command will build 3 docker containers using the localdev compose file.

* Frontend
* Backend
* DB

This should allow you to run backstage-lighthouse locally and be able to view the test data.

### CODE REVIEW

File: packages/backend/package.json [hunk 1]
```diff
@@ -22,7 +22,6 @@
     "@backstage/catalog-model": "^0.9.0",
     "@backstage/cli": "^0.7.7",
     "@backstage/config": "^0.1.4",
-    "@backstage/plugin-app-backend": "^0.3.11",
```
Comment (keyluck) [inline packages/backend/package.json:25]: Do we still need app as a dependency?

Comment (KaemonIsland) [reply packages/backend/package.json:25]: Oh, nope, that should have been removed. Nice catch!

Comment (KaemonIsland) [reply packages/backend/package.json:25]: I misread this commoent.
Yes, from what the docks say, this is only required if we run the frontend and backend together.

From the docs, https://backstage.io/docs/deployment/docker#separate-frontend

> Remove the @backstage/plugin-app-backend and the app package dependency (e.g. app) from packages/backend/packages.json. If you don't remove the app package dependency the app will still be built and bundled with the backend.

Comment (keyluck) [reply packages/backend/package.json:25]: Sorry I wanted to leave a comment on line 34 because I was talking about removing that "app" as a dependency

Comment (KaemonIsland) [reply packages/backend/package.json:25]: I'm not sure actually. I tried not to change anything beyond what the docs said. I'll look into it 👍

**Review by keyluck — APPROVED** [review]

**Pull request merged**